### PR TITLE
Misc typo fixes

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE 
- Bugfix Pull Request

##### COMPONENT NAME
changelogs/fragments/misc_typo_fix.yml
lib/ansible/errors/__init__.py
lib/ansible/modules/cloud/docker/docker_login.py
lib/ansible/modules/utilities/logic/set_fact.py
lib/ansible/plugins/action/nxos_file_copy.py
lib/ansible/plugins/action/synchronize.py
lib/ansible/plugins/callback/dense.py
lib/ansible/plugins/callback/full_skip.py
lib/ansible/plugins/connection/ssh.py
lib/ansible/plugins/inventory/azure_rm.py
lib/ansible/vars/manager.py
test/integration/targets/special_vars/tasks/main.yml

### CODE REVIEW

**Comment by Akasurde** [comment]
cc @Andersson007

**Review by Andersson007 — APPROVED** [review]
funny typos:) seems that it was something wrong with the keyboard:)
shipit

File: lib/ansible/modules/cloud/docker/docker_login.py [hunk 1]
```diff
@@ -169,7 +169,7 @@
             from dockerpycreds.errors import StoreError, CredentialsNotFound
             from dockerpycreds.store import Store
         except ImportError as exc:
-            HAS_DOCKER_ERRROR = str(exc)
+            HAS_DOCKER_ERROR = str(exc)
```
Comment (felixfontein) [inline lib/ansible/modules/cloud/docker/docker_login.py:172]: That's an ugly one. Interesting that no-one ever complained about it...

**Review by felixfontein — APPROVED** [review]
LGTM. I would maybe change `minor_changes` to `bugfixes` in the changelog fragment, though.

File: lib/ansible/modules/cloud/docker/docker_login.py (hunk 1 above)
Comment (felixfontein) [reply lib/ansible/modules/cloud/docker/docker_login.py:172]: Funny thing is, this variable is never used anyway... So not that ugly ;)

**Review by felixfontein — APPROVED** [review]
shipit

**Pull request merged**